packages/devtools_extensions/bin/_build_and_copy.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:io/io.dart';
+import 'package:path/path.dart' as path;
+
+/// Command that builds a DevTools extension and copies the built output to
+/// the parent package extension location.
+/// 
+/// Example usage:
+/// 
+/// dart pub global activate devtools_extensions;
+/// dart run devtools_extensions build_and_copy \
+///  --source=path/to/your_extension_web_app \
+///  --dest=path/to/your_pub_package/extension/devtools 
+class BuildExtensionCommand extends Command {
+  BuildExtensionCommand() {
+    argParser
+      ..addOption(
+        'source',
+        help: 'The source location for the extension flutter web app (can  be '
+            'relative or absolute)',
+        valueHelp: 'path/to/foo/packages/foo_devtools_extension',
+      )
+      ..addOption(
+        'dest',
+        help: 'The destination location for the extension build output (can be '
+            'relative or absolute)',
+        valueHelp: 'path/to/foo/packages/foo/extension/devtools',
+      );
+  }
+
+  static const _sourceKey = 'source';
+  static const _destinationKey = 'dest';
+
+  String get _logPrefix => '[$name]';
+
+  @override
+  final name = 'build_and_copy';
+
+  @override
+  final description =
+      'Command that builds a DevTools extension from source and copies the '
+      'built output to the parent package extension location.';
+
+  @override
+  Future<void> run() async {
+    final source = argResults?[_sourceKey];
+    final destination = argResults?[_destinationKey];
+    if (source == null) {
+      throw ArgumentError(
+        'Missing argument \'$_sourceKey\', which describes the source location '
```

Comment:
Just use double quotations for these strings rather than escaping, here and below.